Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1274,6 +1279,38 @@ def channel_user_list_command():
     return_results(result)
 
 
+def is_bot_in_chat(chat_id):
+    """
+    check if the bot is already in the chat.
+    """
+    url_suffix = f"v1.0/chats/{chat_id}/installedApps?$expand=teamsApp," \
+                 f"teamsAppDefinition&$filter=teamsApp/externalId eq '{BOT_ID}'"
```

Comment:
Please use params instead of explicitly writing the params as part of the suffix.

Reply:
i don't understand exactly what you mean